pom.xml

```diff
@@ -66,8 +66,8 @@
     <gson.version>2.2.4</gson.version>
     <guava.version>13.0.1</guava.version>
     <junit.version>4.11</junit.version>
-    <cask.common.version>0.1.1</cask.common.version>
-    <netty.http.version>0.4.0</netty.http.version>
+    <cask.common.version>0.13.0-SNAPSHOT</cask.common.version>
```

Comment:
Should be 0.12.0, the same as cdap itself. Also, better rename the property to `cdap.common.version`.